### DISCUSSION TITLE
Dedicated Search Page

### DISCUSSION THREAD

**Discussion by SamJUK** [vuejs/vitepress#4327] [Ideas]
I often configure and use Custom Search Engines / Site Shortcuts for quickly navigating technical documentation such as internal company docs / language / framework docs etc.

Its the functionality where in the browser you can enter a keyword into the url bar followed by a search query, and it will directly open the search page on the relevant site for the search query. 

See the following links:
- https://saiankit.medium.com/how-to-setup-custom-search-engines-for-youtube-amazon-etc-in-chrome-brave-fdfd08dd9aa4
- https://support.google.com/chrome/answer/95426?hl=en-GB&co=GENIE.Platform%3DDesktop

For this to work, the ability to populate a search element by query parameters needs to be possible. 

Potential Implementations I see, are either populating the current search component in the header on page load. Or introducing a custom search page that displays the search components as a standalone element.

![image](https://github.com/user-attachments/assets/3726991d-e2f7-4d18-bc21-979035d90b42)

**Comment by brc-dd** [comment]
Track and upvote #2855 and #2983. They are bit low priority for now.
